Engine/Results/CapacityEstimate.cs

```diff
@@ -154,7 +152,7 @@ public void UpdateMarketCapacity(bool forceProcess)
                 var percentageOfHoldings = buyingPowerUsed / totalPortfolioValue;
 
                 var scalingFactor = Math.Max(percentageOfSaleVolume, percentageOfHoldings);
-                var dailyMarketCapacityDollarVolume = smallestAsset.MarketCapacityDollarVolume / days;
+                var dailyMarketCapacityDollarVolume = smallestAsset.MarketCapacityDollarVolume / smallestAsset.Trades;
```

Comment:
nit - variable names are off

Think the change makes sense though 👍🤔, it's the average dollar volume per trade